docs/publishers/config.rst

```diff
@@ -418,6 +433,23 @@ loads.
     The :option:`focus`
     setting is currently still experimental and may change in the future.
 
+.. option:: requestConfigFromFrame
+
+  ``Object``. An object that defines **RPC** settings of a parent controlling iframe. 
```

Comment:
How about:

An object with configuration information about an ancestor `iFrame` that should be able to receive and send RPC messages from/to the client.

IMHO, the rest of the paragraph could probably be omitted, but I'll leave that up to you! If you keep it, please change "requests to the parent" to "requests to the targeted frame".

Reply:
I concur